src/oscar/templatetags/reviews_tags.py

```diff
@@ -36,3 +38,7 @@ def may_vote(review, user):
 @register.filter
 def is_review_permitted(product, user):
     return product and product.is_review_permitted(user)
+
+@register.filter
+def get_reviews(product):
```

Comment:
Or maybe we could use custom model manager query instead of template filter? `{{ product.reviews.approved.all|slice:":3" }}`